iOS/DuckDuckGo/AIChat/InputBox/AIChatInputBox.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  AIChatInputBox.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import UIKit
+
+struct ExpandingTextView: UIViewRepresentable {
```

Comment:
I think it'd be better if the AIChatInputBox view was first and I'd consider moving this into a new file as I don't see anything in here that's chat specific really. But don't feel massively strongly.